docs/rancher/cloud-provider.md

```diff
@@ -48,6 +48,25 @@ When spinning up an RKE2 cluster using the Harvester node driver, select the `Ha
 
   ![](assets/rke2-cloud-provider.png)
 
+### Deploying to the K3s Cluster with Harvester Node Driver
+
+- [Generate addon configuration](https://github.com/harvester/cloud-provider-harvester/blob/master/deploy/generate_addon.sh) and put it in K3s VMs `/etc/kubernetes/cloud-config`.
+
+```
+# depend on kubectl to operate the Harvester cluster
+./deploy/generate_addon.sh <serviceaccount name> <namespace>
+```
+
+- Edit K3s cluster YAML.
+    - Set `disable-cloud-provider: true` to disable default k3s cloud provider.
+    - Add `cloud-provider=external` to use harvester cloud provider.
+
+  ![](assets/edit-k3s-cluster-yaml-for-harvester-cloud-provider.png)
```

Comment:
Will editing the cluster YAML after installation be supported? if not probably better to highlight it.
And the preferred steps will be:
- Choose the Kubernetes version to be k3s and click the `Edit as YAML` button to config the K3s cluster YAML:
![image](https://user-images.githubusercontent.com/4569037/154212613-1740beb2-b247-4a10-a01d-481c37b25910.png)
  - ...
  - ...

Reply:
Yes, we can edit cluster YAML after installation. I update the steps. For the first step, users can click `Edit as YAML` for a new cluster or `Edit YAML` for existing clusters.